services/cd-service/pkg/repository/repository.go

```diff
@@ -894,6 +899,25 @@ func (s *State) GetEnvironmentApplicationLocks(environment, application string)
 	}
 }
 
+func (s *State) GetDeploymentMetaData(environment, application string) (deployAuthor, deployTime string, err error) {
```

Comment:
should be:
`/environments/development/applications
/echo/deployment_author`
`/environments/development/applications
/echo/deployment_time_utc`